src/components/Articles.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utills/api";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((data) => setArticles(data));
+  }, []);
+
```

Comment:
let's add a loading state so that we can conditionally render something like "reviews are loading" whilst we wait for the api request to return